test/server/controllers/UserController.test.js

```diff
@@ -50,3 +53,109 @@ describe('/users/signup', () => {
       });
   });
 });
+
+describe('Testing Login feature -Integration testng', () => {
+  it(
+    'should display error message if user logins with empty field',
+    (done) => {
+      chai.request(app)
+        .post(loginUrl)
+        .send({
+          email: '',
+          password: ''
+        })
+        .end((err, res) => {
+          res.status.should.be.eql(422);
+          res.body.should.be.eql({
+            errors: {
```

Comment:
Should this error return only for email or include password as well?

Reply:
Ok. I will fix that.